packages/dev/core/src/Meshes/greasedLineRibbonMesh.ts

```diff
@@ -0,0 +1,561 @@
+import type { Scene } from "../scene";
+import { Quaternion, Vector3 } from "../Maths/math.vector";
+import { Mesh } from "./mesh";
+import { Buffer } from "../Buffers/buffer";
+import type { Nullable } from "../types";
+import type { Node } from "../node";
+import { DeepCopier } from "../Misc/deepCopier";
+import { GreasedLineTools } from "../Misc/greasedLineTools";
+import type { GreasedLineMeshOptions, GreasedLineRibbonOptions } from "./greasedLineBaseMesh";
+import { GreasedLineBaseMesh, GreasedLineRibbonAutoDirectionMode, GreasedLineRibbonFacesMode, GreasedLineRibbonPointsMode } from "./greasedLineBaseMesh";
+
+Mesh._GreasedLineRibbonMeshParser = (parsedMesh: any, scene: Scene): Mesh => {
+    return GreasedLineRibbonMesh.Parse(parsedMesh, scene);
+};
+
+/**
+ * GreasedLine

```

Comment:
If the class is meant to be instantiated by end users, I think we should add a little more doc here.

Reply:
I mentioned in the docs that both `GresedLineMesh` and `GreasedLineRibbonMesh` are recommended to be created by the `CreateGreasedLine` function.

https://doc.babylonjs.com/features/featuresDeepDive/mesh/creation/param/greased_line#create-a-greasedline

It can be instantiated by the user too but it's quite pain in the ass to set all the properties of the class manually.
I've added this comment to both classes:
```javascript
/**
 * GreasedLineRibbonMesh
 * Use the GreasedLineBuilder.CreateGreasedLine function to create and instance of this class.
 */
```

Do you think it's enough?

Reply:
Perfect!